app/helpers/video_papers_helper.rb

```diff
@@ -6,11 +6,11 @@ module VideoPapersHelper
   def get_video_paper_thumbnail(video_paper)
     unless video_paper.video.nil?
       if video_paper.video.thumbnail?
-        image_tag(video_paper.video.thumbnail.url(:thumb))
+        image_tag(video_paper.video.thumbnail.url(:thumb), alt: "Thumbnail")
       else
         thumbnail_url = video_paper.video.generate_signed_thumbnail_url()
         if thumbnail_url
-          "<img src='#{thumbnail_url}' onerror='this.src=\"#{image_path('blank_fallback.png')}\"' onabort='this.src=\"#{image_path('blank_fallback.png')}\"'/>".html_safe
+          "<img src='#{thumbnail_url}' onerror='this.src=\"#{image_path('blank_fallback.png')}\"' onabort='this.src=\"#{image_path('blank_fallback.png')}\"/>".html_safe

```

Comment:
looks like the quote deleted here is needed to close out the `onabort` attribute.

Reply:
Ah I think the test won't catch this unless they are view tests. And even then perhaps the browser just handles this itself.

Reply:
My bad - I was thinking of adding an alt tag to that image url but decided against it and didn't delete everything.  I've repushed the commit with the fix.